_includes/navigation.html

```diff
@@ -36,13 +36,45 @@
         </div>
         <!-- //navbar item -->
 
-        <!-- navbar item -->
-        <div class="navbar-item">
-          <a href="https://gitter.im/grin_community/Lobby" class="navbar-item">
+        <!-- dropdown -->
+        <div class="navbar-item has-dropdown is-hoverable">
+          <a class="navbar-link">
             Chat
           </a>
+          <div class="navbar-dropdown">
+            <a href="https://gitter.im/grin_community/Lobby" class="navbar-item">
+              Lobby
+            </a>
+            <a href="https://gitter.im/grin_community/support" class="navbar-item">
+              General Support
+            </a>
+            <a href="https://gitter.im/grin_community/dev" class="navbar-item">
+              Workgroup: Development
+            </a>
+            <a href="https://gitter.im/grin_community/design" class="navbar-item">
+              Workgroup: UX & design
+            </a>
+
+            <hr class="navbar-divider">
+
+            <a href="https://gitter.im/grin-pool/Lobby" class="navbar-item">
+              grin-pool support
+            </a>
+            <a href="https://gitter.im/vault713/wallet713" class="navbar-item">
```

Comment:
While I respect a lot vault713's work, I think the grin website should not hint at specific community projects. These are better listed in the wiki page.

Reply:
Agree that vault713 (nor any other community project for that matter) should be promoted on the official website.